src/server/game/Miscellaneous/SharedDefines.h

```diff
@@ -1678,7 +1678,7 @@ enum SpellCastResult
     SPELL_FAILED_NO_TELEPORT_FROM_DUNGEON                       = 251,
     SPELL_FAILED_MAX_LEVEL_TOO_LOW                              = 252,
     SPELL_FAILED_CANT_REPLACE_ITEM_BONUS                        = 253,
-    GRANT_PET_LEVEL_FAIL                                        = 254,
+    SPELL_FAILED_GRANT_PET_LEVEL_FAIL                           = 254,
```

Comment:
this enum is extracted from client so your change will be lost next in next patch

Reply:
Fixed. I thought it was a typo 😄